backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/DockerClusterRabbitMQExtension.java

```diff
@@ -62,14 +59,13 @@ public void beforeAll(ExtensionContext extensionContext) throws Exception {
         rabbitMQ2 = DockerRabbitMQ.withCookieAndHostName(RABBIT_2, clusterIdentity, cookie, network);
         rabbitMQ3 = DockerRabbitMQ.withCookieAndHostName(RABBIT_3, clusterIdentity, cookie, network);
 
-        Runnables.runParallel(
-            rabbitMQ1::start,
-            rabbitMQ2::start,
-            rabbitMQ3::start);
+        startDockerRabbits();
     }
 
     @Override
-    public void beforeEach(ExtensionContext context) {
+    public void beforeEach(ExtensionContext context) throws Exception {
+        startDockerRabbits();
```

Comment:
why are you calling this both in beforeEach and beforeAll?

Reply:
because in some tests, we turn off one rabbit to test remaining rabbits still work well, I need to start if stopped those rabbits to serve next tests, make all dockers are up(if some are stopped).